infra/wca_on_rails/staging/s3.tf

```diff
@@ -10,3 +10,9 @@ resource "aws_s3_bucket" "avatars" {
     "Name" = "wca-avatar"
   }
 }
+resource "aws_s3_bucket" "avatars_private" {
```

Comment:
you are declaring the bucket twice, if you want staging and production to have access to the bucket you need to create it in the shared folder, declare it as an output (and in variables.tf under the shared module) and use that output in the staging and production files. I can do it if you want!

Reply:
I would actually like to take you up on this offer. TIA :smile:

Reply:
I'll do it now